web/src/topic_list.ts

```diff
@@ -282,6 +282,12 @@ export function rebuild($stream_li: JQuery, stream_id: number): void {
     active_widgets.set(stream_id, widget);
 }
 
+function scroll_zommed_in_topic_into_view(): void {
+    const $selected_topic = $(".topic-list .topic-list-item.active-sub-filter");
+    const $container = $("#left_sidebar_scroll_container");
+    scroll_util.scroll_element_into_container($selected_topic, $container);
```

Comment:
What happens if you just have channel feed open, rather than a specific topic? I think we'd want to scroll to the top in that case?

Reply:
oh yeah! Missed that case. Updated to scroll to top in that case.

Reply:
117272820130047025278